web_patient/src/components/Forms/AddEditActivityForm.tsx

```diff
@@ -257,51 +259,57 @@ export const AddEditActivityForm: FunctionComponent<IAddEditActivityFormProps> =
                 modeState: {
                     mode: 'addActivitySchedule',
                     activityId: routeActivityId,
-                }
-            }
+                },
+            };
         } else if (routeParamForm == ParameterValues.form.editActivitySchedule) {
             const routeActivityScheduleId = getRouteParameter(Parameters.activityScheduleId);
             console.assert(!!routeActivityScheduleId, 'editActivitySchedule parameter activityScheduleId not found');
-            if(!routeActivityScheduleId) {
+            if (!routeActivityScheduleId) {
                 return defaultViewState;
             }
 
             const editActivitySchedule = patientStore.getActivityScheduleById(routeActivityScheduleId);
             console.assert(!!editActivitySchedule, 'editActivitySchedule activitySchedule not found');
-            if(!editActivitySchedule) {
+            if (!editActivitySchedule) {
                 return defaultViewState;
             }
 
+            const nextTask = patientStore.getNextTaskByActivityScheduleId(
```

Comment:
@jayfo Made few changes here - fetched `nextTask` to render displayedDate and date.

Rest of the changes are because of code formatting.